aidial_sdk/utils/log_config.py

```diff
@@ -9,7 +9,7 @@ class LogConfig(BaseModel):
     formatters = {
         "default": {
             "()": "uvicorn.logging.DefaultFormatter",
-            "fmt": "%(levelprefix)s | %(asctime)s | %(name)s | %(process)d | %(message)s",
+            "fmt": "%(levelprefix)s | %(asctime)s | %(process)d | %(name)s | %(message)s",
```

Comment:
do we have a good reason to change the order? It can be not good changes for the clients who collect logs and will have two other formats

Reply:
Reverted